employee-management/interface-document.md

```diff
@@ -113,21 +133,21 @@
 
 #### Query parameters
 
-|Required|Parameter name|Value (example)|Type|Description
+|Required|Parameter name|Value (example)|Type|Description|
 |:----|:----|:----|:----|:----|
-||first_name|bien|string|Name employee|
-||last_name|danghuu|string|Last name employee|
-||date_of_birth_start|1996-02-15|string|Start Birthdate of employee|
-||date_of_birth_end|1998-12-30|string|End Birthdate of employee|
-||salary_min|1000|number|Min salary of employee|
-||salary_max|50000|number|Max salary of employee|
-||page|3|number|Display employee information page|
+||first_name|bien|string|Employee's first name|
+||last_name|danghuu|string|Employee's last name|
+||date_of_birth_from|1996-02-15|string|Birthdate of Employee|
+||date_of_birth_to|1998-12-30|string|Birthdate of Employee|
+||salary_min|1000|number|Min salary of Employee|
+||salary_max|50000|number|Max salary of Employee|
+||page|3|number|Display Employee information page|
```

Comment:
ghi cái gì vậy? làm gì có chuyện hiển thị ở đây?

Reply:
Dạ không có Display. Em đã sửa lại: The number of current page is 3